src/main/resources/hapi.properties

```diff
@@ -1,7 +1,19 @@
+
+# Adjust this to set the version of FHIR supported by this server. See
+# FhirVersionEnum for a list of available constants.
 fhir_version=DSTU3
```

Comment:
is src/main/resources/hapi.properties the right place to put the properties file so that the user can deploy the WAR file in tomcat, and then update the properties file, and have subsequent deploys not destroy/overwrite the properties file???

Reply:
@seanmcilvenna good question.

Another thing on my mind- I'd like to configure this project to build both a WAR file as well as an executable JAR (with embedded HTTP server) so that it can be run without needing to install in a container.